src/DataWrangling/inpaint_mask.jl

```diff
@@ -140,7 +140,7 @@ Arguments
 - `inpainting`: The inpainting algorithm to use. For the moment, the only option is `NearestNeighborInpainting(maxiter)`, 
                 where an average of the valid surrounding values is used `maxiter` times.
 """
-function inpaint_mask!(field, mask; inpainting=NearestNeighborInpainting(10))
```

Comment:
found the mistery of the crashing

Reply:
damn the maps

Reply:
i think we should merge this and then open a new PR from the same brnach what do you think?